src/main/java/com/melon/portfoliomanager/models/PortfolioItem.java

```diff
@@ -0,0 +1,76 @@
+package com.melon.portfoliomanager.models;
+
+import jakarta.persistence.*;
+
+
+@Entity
+@Table(name = "portfolio_items")
+public class PortfolioItem {
+
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User user;
+
+    @Column(name = "company_name")
+    private String companyName;
+
+    private Double quantity;
```

Comment:
Does column ```quantity``` map a column in the database table ? If yes, I think it is best for consistency to write @Column annotation as well, specifying the name of the column. (otherwise, we are coupling the name of the field of the class to the name of the column in the database table)